DESCRIPTION

```diff
@@ -21,6 +21,7 @@ Imports:
     bundle,
     butcher (>= 0.3.1),
     cli,
+    ellipsis,
```

Comment:
I was surprised I hadn't imported ellipsis before; maybe I should go through and see how `...` are being checked.